api/v1alpha3/virtualmachineimage_types.go

```diff
@@ -200,6 +215,11 @@ type VirtualMachineImageStatus struct {
 
 	// +optional
 
+	// DiskInfo describes the observed disk information for this image.
```

Comment:
Should this make a mention that this is the just the boot disk?

For my education why are we only including the first disk, instead of having this be an array?

Reply:
Except this will include any disk from the image, right?

Reply:
I just realized this is not a slice :( When I saw the original fields included DiskId, I assumed the parent field was a slice. Talking with Adnan about it.

Reply:
There was some confusion and an assumption on my part. Will update this field to be a slice.